content/desktop/release-notes.md

```diff
@@ -24,11 +24,62 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
+## 4.22.0
+
+{{< release-date date="2023-08-11" >}}
+
+{{< desktop-install all=true version="4.22.0" build_path="/117440/" >}}
+
+### Upgrades
+- [Compose v2.21.0](https://github.com/docker/compose/releases/tag/v2.21.0)
+- [Docker Engine v24.0.6](https://docs.docker.com/engine/release-notes/24.0/#2406)
+- [Docker Scout CLI v0.24.1](https://github.com/docker/scout-cli/releases/tag/v0.24.1).
+- Wasm runtimes:
+  - wasmtime, wasmedge revision `d0a1a1cd`.
+  - slight and spin wasm `v0.9.0`.
+
+### New
+
+- Added support for new Wasm runtimes: wws and lunatic.
+- `docker init` now supports ASP.NET
+- Increased performance of exposed ports, for example with `docker run -p`.
```

Comment:
This improvement is only on macOS, so we should either move it to the For Mac section, or keep it here and add "on macOS" here